ghost/admin/app/components/custom-theme-settings/select.hbs

```diff
@@ -8,7 +8,7 @@
                 {{#each @setting.options as |settingOption|}}
                     <option value={{settingOption}} selected={{eq settingOption @setting.value}}>
                         {{settingOption}}
-                        {{#if (eq settingOption @setting.default)}}(default){{/if}}
+                        {{#if (eq settingOption @setting.default)}}{{/if}}
```

Comment:
```suggestion
```

We can probably remove this whole line now as there is nothing in the conditional?